### PR TITLE
Sr draggable updates

### PR DESCRIPTION
**Pull request by sven-ra**

### CODE REVIEW

File: src/utils/userActive.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { onMounted, onUnmounted, ref, watch } from "vue";
+
+export const useUserActive = () => {
```
Comment (kristjanjansen) [inline src/utils/userActive.ts:3]: Go for https://vueuse.org/core/useidle when it works for you

File: src/components/EButtonBackToEvent.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
```
Comment (kristjanjansen) [inline src/components/EButtonBackToEvent.vue:1]: Does it really needs a dedicated component, why not use EButton and control the class from outside? or add another "color" to the button, such as "idle"?

Can not read the inline SVG, it is an arrow?

File: src/components/EDraggable.vue [hunk 3]
```diff
@@ -50,22 +53,10 @@ const { x, y, style, isDragging } = useDraggable(draggableRef, {
 const snappedX = computed(() => Math.round(x.value / tileSize.value));
 const snappedY = computed(() => Math.round(y.value / tileSize.value));
 
-watch(draggable, (newValue, oldValue) => {
-  isMinimised.value = newValue.isMinimised ? newValue.isMinimised : false;
-  order.value = newValue.order;
-  gridPosX.value = newValue.gridPosX ? newValue.gridPosX : 0;
-  gridPosY.value = newValue.gridPosY ? newValue.gridPosY : 0;
-  // @TODO: Why does dis not work?
```
Comment (kristjanjansen) [inline src/components/EDraggable.vue:58]: Not sure. Is it about deep object compare? what is newValue / onValue type? Ref?

File: src/components/EButtonBackToEvent.vue [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+</script>
+<template>
+  <EButton
+    class="EButtonBackToEvent"
+    :class="{ userActive: userActive }"
+    size="xs"
+    color="transparent"
+    el="a"
+  >
+    <!-- @TODO: Implement proper icon system to Elektro -->
```
Comment (kristjanjansen) [inline src/components/EButtonBackToEvent.vue:18]: Will add the tooling asap

File: src/components/EDraggable.vue [hunk 5]
```diff
@@ -156,6 +203,23 @@ function findCoordinates(el: Element, done: () => void) {
 .EDraggable.isDragging {
   z-index: 100;
 }
+.EDraggable.isVideo .titleBar {
```
Comment (kristjanjansen) [inline src/components/EDraggable.vue:206]: Not sure EDraggable has to know anything about video. What does it affect, how can it be abstraced?

File: src/components/EDraggableTitlebar.vue [hunk 6]
```diff
@@ -1,6 +1,8 @@
 <script setup lang="ts">
 type Props = {
   title?: string;
+  userCount?: number;
```
Comment (kristjanjansen) [inline src/components/EDraggableTitlebar.vue:4]: Usercount feels too much hardcoding, can it be just be a slot with any content? It's ok to TODO / refactor

**Review by kristjanjansen — APPROVED** [review]
LGTM aga voiks teha ka walkthrough tomorrow. Teen ikooni-infra asap.

File: src/utils/userActive.ts (hunk 1 above)
Comment (sven-ra) [reply src/utils/userActive.ts:3]: Yeah, was writing this on the plane, so had no access to vueuse.org :).

File: src/components/EDraggable.vue (hunk 3 above)
Comment (sven-ra) [reply src/components/EDraggable.vue:58]: This part is deleted. What lines are you reffering to?

**Pull request merged**